cmd/kube-apiserver/app/options/globalflags_legacyproviders.go

```diff
@@ -0,0 +1,30 @@
+// +build !providerless
```

Comment:
build tag and file name don't match any more...

Reply:
ACK